global.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+
+        <script src="./hamburger.js" defer></script>
+        <link rel="stylesheet" href="./global.css" />
+    </head>
+    <body>
+        <header class="nav">
+            <div class="nav_left">
+                <img
+                    class="nav_logo"
+                    src="./assets/Icons/logo-with-credard.svg"
```

Comment:
The logo image is not showing, please check why and also the hamburger menu is not there anymore

![image](https://user-images.githubusercontent.com/60205816/182897244-0e8283b8-c55f-4e3f-a8d1-32d5ab9912b6.png)